docker-monitoring/README.md

```diff
@@ -0,0 +1,15 @@
+# Docker Monitoring
```

Comment:
This stack can be used to monitor any deployment of `mezod`, regardless of how it was actually set up, right? If so, what do you think about renaming the package from `docker-monitoring` to just `monitoring`.

Reply:
No. It's dedicated to v-kit based on Docker, because:
- logs are scraped from containers (https://github.com/mezo-org/validator-kit/pull/18/files#diff-bc1c2ab1c2d18bf220a04a2fda94ee616b14c85af52d5cb7dfc0a451bf98a24cR12) using proper labels (https://github.com/mezo-org/validator-kit/pull/18/files#diff-089d1c7c9c347f34b1d7df58f7fe54fc8fdf4965b0d591992d5c08b39dd7bf24R117)
- metrics are scraped from containers and statically defined targets (checkout `docker-monitoring/configs/prometheus/prometheus.yaml`)

It is possible to use it for native v-kit, but it requires many adjustments.

Reply:
Ok, I see. Thanks for clarification.